index.bs

```diff
@@ -28,11 +28,6 @@ Ignored Vars: e
 </pre>
 
 <pre class="anchors">
-urlPrefix: https://html.spec.whatwg.org/multipage/; spec: HTML
-    text: StructuredClone; urlPrefix: infrastructure.html; type: abstract-op
-    text: Transfer; url: infrastructure.html#transfer-abstract-op; type: abstract-op
-    text: WebSocket; url: comms.html#websocket; type: interface
-    text: transferable objects; url: infrastructure.html; type: dfn
```

Comment:
I pulled this and some other changes that are unrelated to teeing out into https://github.com/whatwg/streams/commit/0dc890351d298e288161c991582d998a9d5e5f51 (committed to master)